package.json

```diff
@@ -7,11 +7,13 @@
     "babel-core": "6.8.0",
     "babel-eslint": "4.1.7",
     "babel-loader": "6.2.4",
+    "babel-plugin-add-module-exports": "0.2.1",
```

Comment:
Good to know that, thanks! So far I believe only two plugins use dops-components, and they all use babel to build finalized JS that's going to be published. So they have dev dependencies installed, they don't use Node in production.
But you're right, we need to work on that in the future.